homeassistant/components/switcher_kis/switch.py

```diff
@@ -36,6 +41,24 @@
     "auto_off_set": ATTR_AUTO_OFF_SET,
 }
 
+SERVICE_SET_AUTO_OFF_NAME = "set_auto_off"
+SERVICE_SET_AUTO_OFF_SCHEMA = vol.Schema(
```

Comment:
Please inline the schema when registering the service and only pass a dict with voluptuous validators. The entity service helper will generate a entity service schema for us automatically.

https://developers.home-assistant.io/docs/dev_101_services#entity-services

Reply:
Fixed